CodeBattleCPP/CodeBattleCppLibrary/BoardElement.cpp

```diff
@@ -141,6 +165,12 @@ BoardElement BoardElementFromString(const std::string &str)
         return BoardElement::OTHER_HERO_PIPE_RIGHT;
     if (str == kOTHER_HERO_SHADOW_DIE)
         return BoardElement::OTHER_HERO_SHADOW_DIE;
+    if (str == kOTHER_HERO_SHADOW_DRILL_LEFT)
+        return BoardElement::OTHER_HERO_SHADOW_DRILL_LEFT;
+    if (str == kOTHER_HERO_SHADOW_DRILL_RIGHT)
+        return BoardElement::OTHER_HERO_SHADOW_DRILL_RIGHT;
+    if (str == kOTHER_HERO_SHADOW_FALL_LEFT)
+        return BoardElement::OTHER_HERO_SHADOW_FALL_LEFT;
```

Comment:
kOTHER_HERO_SHADOW_FALL_RIGHT seems missed

Reply:
Nice catch! Thank you